src/Prismic/Dom/Date.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Prismic\Dom;
+
+class Date
+{
+    /**
+     * Returns the date as a DateTime object
+     *
+     *
+     * @param string $date The date as a string
+     *
+     * @return \DateTime The DateTime object representing the date
+     */
+    public static function asDate($date)
+    {
+        return new \DateTime($date);
```

Comment:
I believe you should use: 
`DateTimeImmutable::createFromFormat(DateTime::ISO8601, $date)`

Reply:
And what about `Timestamp` ?